src/components/accession2/edit/EditLocationModal.tsx

```diff
@@ -89,6 +89,7 @@ export default function EditLocationModal(props: EditLocationModalProps): JSX.El
       <Grid item xs={12} textAlign='left'>
         <Grid item xs={12}>
           <StorageLocationSelector
+            id='location'
```

Comment:
not sure if you want to distinguish between this use-case and the other.. if so, maybe call it `edit-location` .. etc.

Reply:
i guess the page url helps distinguish here,, since these flows aren't modals

Reply:
Good practice anyways!